src/main/resources/public/js/app.js

```diff
@@ -9,11 +9,20 @@ $( document ).ready(function() {
    });
 });
 
-
 var state = "fire";
 var lastClickAcross;
 var lastClickDown;
 
+function startGame(mode) {
+	if (mode == "hard") {
+		
```

Comment:
Does this happen after we get a blank model or before? If we already have a model, we could probably set the flag here, otherwise, we should set a flag for later use. maybe not your task, but if you know which one, it would help